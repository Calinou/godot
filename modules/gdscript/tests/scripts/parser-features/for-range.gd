func test():
	for i in range(5):
		print(i)

	print()

	# Equivalent to the above `for` loop:
	for i in 5:
		print(i)

	print()

	for i in range(1, 5):
		print(i)

	print()

	for i in range(1, -5, -1):
		print(i)

	for i in [2, 4, 6, -8]:
		print(i)

	for i in [true, false]:
		print(i)

	for i in [Vector2i(10, 20), Vector2i(30, 40)]:
		print(i)

	for i in "Hello Unic√¥de world!":
		print(i)
