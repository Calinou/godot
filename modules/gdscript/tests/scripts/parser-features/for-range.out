GDTEST_OK
0
1
2
3
4

0
1
2
3
4

1
2
3
4

1
0
-1
-2
-3
-4
2
4
6
-8
True
False
(10, 20)
(30, 40)
H
e
l
l
o
 
U
n
i
c
Ã´
d
e
 
w
o
r
l
d
!
